studio/components/interfaces/Auth/AuthProvidersForm/ProviderForm.tsx

```diff
@@ -19,36 +20,40 @@ export interface ProviderFormProps {
 
 const ProviderForm = ({ provider }: ProviderFormProps) => {
   const [open, setOpen] = useState(false)
-  const { authConfig, ui } = useStore()
-  const { project: selectedProject } = useProjectContext()
-  const { ref } = useParams()
+  const { ui } = useStore()
+  const { ref: projectRef } = useParams()
+  const { data: config } = useAuthConfigQuery({ projectRef })
+  const { mutate: updateAuthConfig } = useAuthConfigUpdateMutation()
```

Comment:
use isUpdatingConfig here instead of isSubmitting in the Form component below

```suggestion
  const { mutate: updateAuthConfig, isLoading: isUpdatingConfig } = useAuthConfigUpdateMutation()
```